src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -75,7 +75,6 @@ internal static TypeSymbol CopyTypeCustomModifiers(TypeSymbol sourceType, TypeSy
             // NOTE: ref is irrelevant here since we are just encoding/decoding the type out of the signature context
             ImmutableArray<bool> flags = CSharpCompilation.DynamicTransformsEncoder.EncodeWithoutCustomModifierFlags(destinationType, refKind);
             TypeSymbol typeWithDynamic = DynamicTypeDecoder.TransformTypeWithoutCustomModifierFlags(sourceType, containingAssembly, refKind, flags);
-            // PROTOTYPE: Should we use NativeIntegerTypeDecoder here?
```

Comment:
>// PROTOTYPE: Should we use NativeIntegerTypeDecoder here? [](start = 12, length = 58)

It is not clear why this comment is removed. It doesn't look like the encoder is called, but I think it should be called

Reply:
[https://github.com/dotnet/roslyn/issues/42500](https://github.com/dotnet/roslyn/issues/42500)

---
In reply to: [393306528](https://github.com/dotnet/roslyn/pull/42419#discussion_r393306528) [](ancestors = 393306528)